frontend/app/src/App.tsx

```diff
@@ -1061,6 +1061,11 @@ export class App extends PureComponent<Props, State> {
         newDialog,
         sessionEvent.scriptCompilationException?.message ?? "No message"
       )
+    } else if (sessionEvent.type === "scriptChangedOnDisk") {
```

Comment:
If we trigger the run on save rerun here via the `scriptChangedOnDisk`, isn't this also showing the script change prompt in the top right corner?